docs/notification-cleanup-monitoring.sql

```
@@ -0,0 +1,235 @@
+-- Adjust HAVING threshold based on your batch size (e.g., if batch=100, use 10+)
+SELECT 
+	"NotificationSent",
+	COUNT(*) as count,
+	MIN("Id") as min_id,
+	MAX("Id") as max_id,
+	ROUND(100.0 * COUNT(*) / (SELECT COUNT(*) 
+		FROM correspondence."CorrespondenceNotifications"
+		WHERE "Altinn2NotificationId" IS NOT NULL 
+		  AND "SyncedFromAltinn2" IS NOT NULL
+		  AND "NotificationSent" < '2026-04-25'::timestamptz), 4) as percent_of_total
+FROM correspondence."CorrespondenceNotifications"
```

SQLFluff (4.2.2)

[error] Duplicate table alias '"CorrespondenceNotifications"'. Table aliases should be unique.

(AL04)